#!/usr/bin/env python
PACKAGE = "roomac_arm"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_change_threshold", double_t, 0, "", 0.01, 0, 0.1)
gen.add("max_speed", double_t, 0, "", 0.5, 0, 2.5)

gen.add("interpolate_movement", bool_t, 0, "", True)
gen.add("interpolation_frequency", double_t, 0, "", 10.0, 0, 50.0)

gen.add("wait_time_scaling", double_t, 0, "", 1.0, 0.0, 1.0)

exit(gen.generate(PACKAGE, "roomac_arm", "ArmController"))